ValveKeyValue/ValveKeyValue/Deserialization/KV1BinaryReader.cs

```diff
@@ -17,19 +17,19 @@ public KV1BinaryReader(Stream stream, IVisitationListener listener)
                 throw new ArgumentException("Stream must be seekable", nameof(stream));
             }
 
-            this.stream = stream;
-            this.listener = listener;
-            reader = new BinaryReader(stream);
+            _stream = stream;
+            _listener = listener;
+            _reader = new BinaryReader(stream);
         }
 
-        readonly Stream stream;
-        readonly BinaryReader reader;
-        readonly IVisitationListener listener;
-        bool disposed;
+        private readonly Stream _stream;
```

Comment:
Can you revert these changes please?

It makes for an unnecessarily hard diff to read.

Reply:
Done